geniza/corpus/views.py

```diff
@@ -76,7 +76,14 @@ def get_queryset(self):
                     .also("score")
                 )  # include relevance score in results
 
-            documents = documents.order_by(self.solr_sort[search_opts["sort"]])
+            documents = documents.order_by(
+                self.solr_sort[search_opts["sort"]]
+            ).facet_field("type", exclude="type", sort="value")
+            # exclude type filter when generating counts
+
+            # filter by type if specified
+            if search_opts["doctype"]:
+                documents = documents.filter(type=search_opts["doctype"], tag="type")
```

Comment:
This only seems to work when I filter by doctype "Letter". Were you experiencing that too or have I misconfigured something? The only reason I can think of for this is that it doesn't contain multiple words…

Reply:
oh, good catch — it needs to be quoted since some of these are multiple words

Reply:
Something like this?
```suggestion
                documents = documents.filter(type=f'"{search_opts["doctype"]}"', tag="type")
```